ajax/libs/focusable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "focusable",
+  "filename": "focus-element-overlay.min.js",
+  "version": "1.0.0",
```

Comment:
If you would like to add the library via single package.json, please remove this line.